monero-outreach-docs/translations/zh-hant/quickfacts/quickfacts_zh-hant.md

```diff
@@ -109,13 +105,11 @@ _Monero 的區塊獎勵會緩慢下降直到 2022 年五月，屆時將有 一
 
 雖然 Monero 已經可以在全世界使用，社群開發者仍然還有一些令人興奮的目標，都是為了讓 Monero 更加安全、更好的隱私保護與更容易使用的功能。有些目標預計很快就能完成：
 
-防彈協定：[這項開發](https://getmonero.org/2017/12/07/Monero-Compatible-Bulletproofs.html)可以讓交易在區塊鏈上儲存的效率提升，並帶來更快的交易速度。
-
-**Kovri：**這會是一個隱私保護功能的升級，為 Monero 交易加上第四層保護。 Kovri 使用**[garlic-encryption](https://getmonero.org/resources/moneropedia/garlic-encryption.html)**(就像是 Tor on steroids) 用來遮蔽發送者與接收者在網路上的 IP 位址。如此一來，在 Monero 上的交易每一層面包括發送者、接收者、交易金額和交易本身(使用 Kovri)將都會得更有隱私且更加安全。
+**Kovri：** 這會是一個隱私保護功能的升級，為 Monero 交易加上第四層保護。 Kovri 使用 **[garlic-encryption](https://getmonero.org/resources/moneropedia/garlic-encryption.html)** (就像是 Tor on steroids) 用來遮蔽發送者與接收者在網路上的 IP 位址。如此一來，在 Monero 上的交易每一層面包括發送者、接收者、交易金額和交易本身(使用 Kovri)將都會得更有隱私且更加安全。
 
-**硬體錢包：**Trezor Model T 硬體錢包已經將 Monero 列入**[支援清單](https://trezor.io/coins/#XMR)**，而**[Ledger 硬體錢包](https://support.ledgerwallet.com/hc/en-us/articles/360006352934-Monero-XMR-Advanced)**已可以在 Monero 的官方 GUI 錢包中使用。而另外 Monero社群更成功募資支持了一個團隊以**[打造 Kastelo 開源硬體錢包](http://kastelo.org/)**，系統將基於 Monero ethos 上開發。這些硬體錢包都完全支援 Monero 的交易。
+**硬體錢包：** 熱門的 Ledger 和 Trezor 硬體錢包都正在加入 Monero（Ledger 已可以在 Monero 官方的 GUI v0.12.3 和 CLI v0.12.2 錢包中使用），而另外 Monero社群更成功募資支持了一個團隊以 **[打造 Kastelo 開源硬體錢包](http://kastelo.org/)** ，系統將基於 Monero ethos 上開發。這些硬體錢包都完全支援 Monero 的交易。
```

Comment:
We also made an addition to the last sentence of this paragraph. Added some missing words and made it a little more understandable. This is the last sentence:

"Both Ledger and Trezor hardware wallet options were anticipated to be operational by the end of 2018."

You can take a look at #99 and see the change in the last sentence of the paragraph.

Reply:
Revised.